FileShareService.DesktopClient.Core/AuthProvider.cs

```diff
@@ -91,6 +90,7 @@ public async  Task<string> GetToken()
             var cancellationSource = new CancellationTokenSource(TimeSpan.FromMinutes(2));
             TokenCacheHelper.EnableSerialization(publicClientApplication.UserTokenCache);         
             var accounts = (await publicClientApplication.GetAccountsAsync()).ToList();
+            var TempToken = CurrentAccessToken;
```

Comment:
Please change the capitalisation of `TempToken` to match the convention for local variables

Reply:
Resolved